code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm

```diff
@@ -0,0 +1,212 @@
+	..()
+	return TRUE
+
+/datum/reagent/medicine/C2/theophylline/on_mob_end_metabolize(mob/living/L)
+	if(current_cycle > 20)
+		L.Sleeping(200)
+	..()
+
+/******TOXIN******/
+
+#define CORTI_WEAKRATE	0.005
+
+/datum/reagent/medicine/C2/corticoline //made up chem from Corticosteriods, used to treat POISON ivy wink wink
+  name = "Corticoline"
+  description = "An antitoxin that temporarily weakens the user, making them susceptible to other forms of damage. Weakness and toxin healing scales with exposure."
+  overdose_threshold = 11 // Scales every damage so naturally quite abusable. Luckily the OD purges the chem which resets the damage mod *wink*
+  var/reset_mods = 0 //amount, not bool
+
+/datum/reagent/medicine/C2/corticoline/on_mob_life(mob/living/carbon/human/M)
+	var/maths = CORTI_WEAKRATE * current_cycle
+	reset_mods += maths
+	var/datum/physiology/phis = M.physiology
+	phis.brute_mod += maths
+	phis.burn_mod += maths
+	phis.oxy_mod += maths
+	phis.stamina_mod += maths
+	M.adjustToxLoss(round(reset_mods*-1000,0.01)) //Math is fun!
+	..()
+	return TRUE
+
+/datum/reagent/medicine/C2/corticoline/on_mob_delete(mob/living/carbon/human/M)
+	var/datum/physiology/phis = M.physiology
+	phis.brute_mod = max(phis.brute_mod - reset_mods, 1)
+	phis.burn_mod = max(phis.burn_mod - reset_mods, 1)
+	phis.oxy_mod = max(phis.oxy_mod - reset_mods, 1)
+	return ..()
+
+/datum/reagent/medicine/C2/corticoline/overdose_process(mob/living/carbon/human/M)
+	var/maths = reset_mods*3
+	M.adjustBruteLoss(maths)
+	M.adjustFireLoss(maths)
+	M.adjustOxyLoss(maths)
+	..()
+	M.reagents.del_reagent(type)
+	return TRUE
+
+#undef CORTI_WEAKRATE
+
+/datum/reagent/medicine/C2/palletta //made up chem, it's a PALLETTE cleanser hehe
+	name = "Palletta"
+	description = "An antitoxin that scales with the more chems in the body as well as purges chems (including itself). Causes loss of breath."
+
+/datum/reagent/medicine/C2/palletta/on_mob_life(mob/living/carbon/human/M)
+	var/total_v = 0
+	for(var/r in M.reagents.reagent_list)
+		var/datum/reagent/the_reagent = r
+		total_v += the_reagent.volume
+	M.adjustToxLoss(total_v*-0.1)
+	for(var/datum/reagent/R in M.reagents.reagent_list)
+		M.reagents.remove_reagent(R.type,R.volume*0.1)
+		M.losebreath += total_v*0.1
+	..()
+
+/******COMBOS******/
+/datum/reagent/medicine/C2/synthflesh
+	name = "Synthflesh"
+	description = "Has a 100% chance of instantly healing brute and burn damage at the cost of toxicity (75% of damage healed). Touch application only."
+	reagent_state = LIQUID
+	color = "#FFEBEB"
+
+/datum/reagent/medicine/C2/synthflesh/reaction_mob(mob/living/M, method=TOUCH, reac_volume,show_message = 1)
+	if(iscarbon(M))
+		var/mob/living/carbon/Carbies = M
+		if (Carbies.stat == DEAD)
+			show_message = 0
+		if(method in list(PATCH, TOUCH))
+			var/harmies = min(Carbies.getBruteLoss(),Carbies.adjustBruteLoss(-1.25 * reac_volume)*-1)
+			var/burnies = min(Carbies.getFireLoss(),Carbies.adjustFireLoss(-1.25 * reac_volume)*-1)
```

Comment:
I _will_ come to your house with a printed copy of these variable names and _**feed it to you.**_ Also, see palletta for potential issues with toxin conversion.

Reply:
will remaths palletta.